File: Trabajo #5/Python/Trabajo#6.py
print ("Registradora almacen")
na=int(input("Ingrese el numero de articulos "))
va=int(input("Ingrese el valor de los articulos "))
v=int(input("Desea ingresar otro articulo? \n digite 1 si o 2 para no "))
st=na*va
art=2
while v==1:
    na=int(input("Ingrese para el articulo [art]\n numero de articulos: "))
    va=int(input("Valor de los articulos "))
    v=int(input("Desea ingre otro articulo? \n digite 1 si o 2 para no "))
    st= st + (va*na)
    art+=1
v= int(input("Cual es su forma de pago? \nDigite 1 credito o 2 efectivo"))
iva=st*0.16
print("El valor total de articulos ingresados fue: ",art-1)
if(v==2):
    print(f"su compra fue de {st} \niva = {iva}\nTotal = ",st+iva)
else:
    print(f"su compra fue de {st} \niva = {iva}\nDescuentos: ",(st+iva)*0.1,"\nTotal = ",st+iva-((st+iva)*0.1))

File: Trabajo #5/C++/Trabajo#6.cpp
#include <iostream>

using namespace std;
int va, na, iva, st, v, des;
int main(){
	cout<<"Ingrese el numero de articulos: ";
	cin>>na;
	cout<<"Ingrese el valor de los articulos: ";
	cin>>va;
	cout<<"Desea ingresar otro articulo?"<<"Digite 1 si o 2 para no ";
	cin>>v;
	st=va*na;
	while (v==1){
		cout<<"Ingrese el numero de articulos: ";
		cin>>na;
		cout<<"Ingrese el valor de articulos: ";
		cin>>va;
		st= st+(va*na);
		cout<<"Desea ingresar otro articulo? "<<"Digite 1 si o 2 para no ";
		cin>>v;
	}
	iva=st*0.16;
	cout<<"Cual es su forma de pago? "<<"Digite 1 credito o 2 efectivo ";
	cin>>v;
	if(v==1){
		cout<<"Su compra fue de: "<<st<<"\nIva = "<<iva<<" \nTotal= "<<st+iva;
	}
	else{
		cout<<"Su compra fue de: "<<st<<" \nIva= "<<iva<<"\nDescuento: "<<(st+iva)*0.16;
	}
		
	}
